src/index.html

```diff
@@ -67,9 +67,9 @@
   <body>
```

Comment:
With the addition of `zh-Hans`, I think we should add `rel="alternate"` `link` tags with `hreflang`, but I'm not sure if that's appropriate.

@mattbasta do you have any insight here?

https://developers.google.com/search/blog/2013/04/x-default-hreflang-for-international-pages

Reply:
I don't have context on how your site is set up, but I could look. What you mentioned is for sites that provide each localized version of the content on different URLs. If you're doing that, you'll want the tags, yes. If you're just looking at browser locale and maybe a locale cookie, you can just do with a HTML lang attribute on the root tag.

Reply:
this is a static site hosted on GitHub pages. I don't have the ability to inspect the `Accept-Language` header. This PR sets things up so `en` the default locale has un-prefixed URLs and the `zh-Hans` pages will be prefixed in the URL.

So https://www.artichokeruby.org/ -> https://www.artichokeruby.org/zh-hans/ and https://www.artichokeruby.org/install/ -> https://www.artichokeruby.org/zh-hans/install/.

I imagine I'll want:

- the `hreflang` tags
- A language selector in the navbar
- Maybe autodetection of browser preferred lang with JS + a cookie and redirect?